src/containers/Home/FavList.js

```diff
@@ -0,0 +1,181 @@
+  static props = {
+    visibility: false,
+    place: null,
+    favChoosed: '',
+  };
+
+  // one of the list title selected
+  onItemSelected(count) {
+    if (this.props.favChoosed !== this.props.favouriteList[count].title) {
+      this.props.doSetFavChoosed(this.props.favouriteList[count].title);
+    }
+    this.props.doSetFavMode(true);
+    this.props.doSetFav(false);
+  }
+
+  modeSwitch() {
+    this.hideMenu();
+    if (this.props.favMode) {
+      this.props.doSetFavMode(false);
+    }
+    this.props.doSetFavChoosed('');
+  }
+
+  hideMenu() {
+    if (this.props.visibility) {
+      this.props.doSetFav(false);
+    }
+  }
+
+  handleBackButton() {
+    this.hideMenu();
+    return false;
+  }
+
+  renderItems(count: number) {
+    const { title } = this.props.favouriteList[count];
+    return (
+      <TouchableHighlight key={count} onPress={() => this.onItemSelected(count)}>
+        <View
+          style={{
+              flex: 1,
+              height: 60,
+              flexDirection: 'row',
+              backgroundColor: title === this.props.favChoosed ? '#D6EBFF' : '#FFFFFF',
+          }}
+        >
+          <View style={{ flex: 5, alignItems: 'flex-start', justifyContent: 'center' }}>
+            <Text style={{
+              fontSize: 20,
+              marginLeft: 20,
+            }}
+            >{title}
+            </Text>
+          </View>
+          <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+            <Text style={{
+              fontSize: 20,
+              marginRight: 20,
+            }}
+            >{this.props.favouriteList[count].items.length}
+            </Text>
+          </View>
+        </View>
+      </TouchableHighlight>
+    );
+  }
+
+  renderFavItems() {
+    const items = [];
+    for (let i = 0; i < this.props.favouriteList.length; i += 1) {
+      items.push(this.renderItems(i));
+    }
+    return items;
+  }
+
+  renderHeader() {
+    return (
+      <View
+        style={{
+            flexDirection: 'row',
+            width: '100%',
+            height: 50,
+            alignItems: 'flex-start',
+            backgroundColor: this.props.favMode ? colors.subAccentColor2 : colors.subAccentColor,
+            }}
+      >
+        <Text style={{
+          color: this.props.favMode ? '#808080' : '#FFFFFF', fontSize: 15, marginLeft: 20, marginTop: 13,
+          }}
+        >
+        Choose from favourite list
+        </Text>
+        <View
+          style={{
+            flex: 1, alignItems: 'flex-end', justifyContent: 'center',
+          }}
+        />
+        {this.props.favMode ? this.renderBACKButton() : null}
+      </View>
+    );
+  }
+
+  renderBACKButton() {
+    return (
+      <TouchableHighlight
+        onPress={() => this.modeSwitch()}
+        style={{
+          width: '15%', height: '50%', marginRight: 20, marginTop: 15,
+        }}
+      >
+        <Text style={{ textAlign: 'center', fontSize: 15 }}>BACK</Text>
```

Comment:
style should put in its own style file

Reply:
if some like only one phrase or two phrase also need to put into style file meh?